index.html

```diff
@@ -61,22 +78,22 @@
 				// },
 				//Phường 4
 				//Example
-				// {
-				// 	type: "point",
-				// 	longitude: 106.80378931111271, 
-				// 	latitude: 10.870369163501929,
-				// 	Name: "Ghế ngồi",
-				// 	Location: "trong khuông viên trường ĐH CNTT",
-				// 	symbol: {
-				// 		type: "simple-marker",
-				// 		color: [153, 51, 51],
-				// 		outline: {
-				// 			color: [255, 255, 255],
-				// 			width: 1
-				// 		}
-				// 	},
-				// 	popupTemplate: point_template_chair
-				// },
+				{
+					type: "point",
+					longitude: 106.68127715688188, 
+					latitude: 10.774075083216818,
+					Name: "Bệnh viện Bình Dân",
+					Location: "Địa chỉ: 371 Điện Biên Phủ, Phường 4, Quận 3, Hồ Chí Minh 700000, Việt Nam",

```

Comment:
You can show the text "Địa chỉ" in your popup like HTML. Please remove it.

Reply:
So, what is content in location ?

Reply:
chỗ này em chỉ cần xoá cái chữ `Địa chỉ:` nha @wsoft416